contact.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Contact </title>
+    <link rel="icon" href="./Photo/photo4.png" type="image/icon type">

```

Comment:
În loc de `image type` ar trebui [tipul de fișier](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types). În cazul tău probabil `image/png`

Reply:
```
```suggestion
    <link rel="icon" href="./Photo/photo4.png" type="image/png">
```